aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -672,4 +680,64 @@ TORCH_IMPL_FUNC(addmv_out_cuda)(const Tensor &self, const Tensor &mat, const Ten
   }
 }
 
+
+void _int_mm_out_cuda(const Tensor& self, const Tensor& mat2, Tensor& result) {
+  TORCH_CHECK(self.size(0) > 16, "self.size(0) needs to be greater than 16, but got ", self.size(0));
```

Comment:
cublasLt has a lot of annoying limitations aka bugs (like inputs have to be contiguous or transposed, or it errors out), the ones that we know about are collected in regular addmm in use_cublasLt check, can you double check that you are either checking for them or test that corner case things actually work?

Reply:
Yep, and for int8 there's a whole other set of similar features. I'm still uncovering some of them. Will create a separate check.